tests/jeos/build_key.pm

```diff
@@ -14,18 +14,25 @@ use strict;
 use warnings;
 use base "opensusebasetest";
 use testapi;
+use version_utils "is_opensuse";
 
 sub run {
     # check that suse-build-key is installed
     my $zypper_output = script_output('zypper search suse-build-key');
```

Comment:
Why not just `assert_script_run('rpm -qi suse-build-key')` or `openSUSE-build-key` depending on `is_opensuse`?

This regex use seems superfluous to me.